packages/aws-library/requirements/_test.in

```diff
@@ -11,6 +11,8 @@
 # testing
 coverage
 faker
+fastapi
```

Comment:
Why do you need these dependencies? I do not see any changes in the tests of this package. Did you miss something?

Reply:
yep. s3 helpers in pytest-simcore